### PR TITLE
Add ip,port args and url-logs command

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/23629

## Description
- Added ip and port arguments to cdl-query-threat-logs and cdl-query-traffic-logs commands
- Added cdl-query-url-logs command

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 1]
```diff
@@ -75,10 +75,7 @@ configuration:
   name: incidentType
   required: false
   type: 13
-- additionalinfo: |-
```
Comment (reutshal) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:78]: Why did you remove the additional info?

File: Packs/CortexDataLake/Integrations/CortexDataLake/README.md [hunk 2]
```diff
@@ -1,76 +1,46 @@
...
- - For authentication: 
-   - `oproxy.demisto.ninja`
-   - `api.paloaltonetworks.com `
- - For API requests, one of the following:
-   - US: `api.us.cdl.paloaltonetworks.com`
-   - EU: `api.nl.cdl.paloaltonetworks.com`
-
-## Fetched Incidents Data
-Fetches Firewall threat logs as incidents
-
```
Comment (reutshal) [inline Packs/CortexDataLake/Integrations/CortexDataLake/README.md:45]: Why was this section removed?

**Comment by ronykoz** [comment]
@barchen1 what's the status here?

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml (hunk 1 above)
Comment (barchen1) [reply Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:78]: I think we are not using this key anymore cause it made sdk-validate fail on scheme error.

File: Packs/CortexDataLake/Integrations/CortexDataLake/README.md (hunk 2 above)
Comment (barchen1) [reply Packs/CortexDataLake/Integrations/CortexDataLake/README.md]: done

**Comment by barchen1** [comment]
@kirbles19 Can you please go over the yml and Readme file?
Only for the new arguments (ip, port) in cdl-query-threat-logs and cdl-query-traffic-logs commands and the new command "cdl-query-url-logs".

**Comment by barchen1** [comment]
> @barchen1 what's the status here?

Ready.
just need @kirbles19 approval

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py [hunk 3]
```diff
@@ -424,11 +478,32 @@ def build_where_clause(args: dict) -> str:
         'file_sha_256': 'file_sha_256',
         'file_name': 'file_name',
     }
+    if args.get('ip') and (args.get('source_ip') or args.get('dest_ip')):
+        raise DemistoException('Error: "ip" argument cannot appear with either "source_ip" nor "dest_ip"')
+
+    if args.get('port') and args.get('source_port') or args.get('port') and args.get('dest_port'):
```
Comment (ronykoz) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py:484]: use the same condition logic as in line 481

Comment (barchen1) [reply Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py]: done

**Comment by richardbluestone** [comment]
Docs reviewed and updated

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py [hunk 4]
```diff
@@ -794,6 +801,7 @@ def build_query(args, table_name):
     limit = args.get('limit', '5')
     where += f' AND {timestamp_limitation}' if where else timestamp_limitation
     query = f'SELECT {fields} FROM `firewall.{table_name}` WHERE {where} LIMIT {limit}'
+    print(query)
```
[the remark this answers is not in the document]
Comment (barchen1) [reply Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py:804]: yes
